Task_TheGridProblem/index.html

```diff
@@ -1,11 +1,23 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <title>Grid Problem</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
```

Comment:
Consider adding a media query in the index.css file to specify different styles for different screen sizes.